presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -415,7 +311,7 @@ as a part of SQL query by appending name with the table.
        0     | s3://my-bucket/ctas_nation/data/9f889274-6f74-4d28-8164-275eef99f660.parquet | PARQUET     |           25 |               1648 | {1=52, 2=222, 3=105, 4=757} | {1=25, 2=25, 3=25, 4=25} | {1=0, 2=0, 3=0, 4=0} |  NULL            | {1=0, 2=ALGERIA, 3=0, 4= haggle. careful} | {1=24, 2=VIETNAM, 3=4, 4=y final packaget} | NULL         | NULL          | NULL
 
 Time Travel
-------------------------
+-----------
```

Comment:
This comment is for ln 323
At this moment, the Schema Evolution has not been talked about.

Reply:
And for line 355, you mentioned we just created the ctas table, but this was not mentioned anywhere above. It might be good to have basic DDL and DML sections before the SELECT queries.

Reply:
> This comment is for ln 323 At this moment, the Schema Evolution has not been talked about.

Ok, I will shift Schema Evolution before Time Travel Section in the document

Reply:
> And for line 355, you mentioned we just created the ctas table, but this was not mentioned anywhere above. It might be good to have basic DDL and DML sections before the SELECT queries.

@yingsu00 In the `Example Queries` section of the same there first query is CTAS, and all the further queries are done on the same table. Please let me know if anything is missing here.